start.py

```diff
@@ -144,6 +145,31 @@ def set_openpype_global_environments() -> None:
     if "QT_AUTO_SCREEN_SCALE_FACTOR" not in os.environ:
         os.environ["QT_AUTO_SCREEN_SCALE_FACTOR"] = "1"
 
+    # --- Set environment variables to vendorized binaries(FFmpeg and OIIO) ---
+    # TODO add validations of existing binaries
+    # Prepare path to bin vendor directory
+    bin_vendor_dir = os.path.join(
```

Comment:
This will need to be changed for running from build. You should be combining the path with sys.executable instead of the __file__

Reply:
Added `BUILD_ROOT` global variable which is defined at the beginning of `start.py`. Definition is based on condition `getattr(sys, 'frozen', False)`. The variable was replaced at places using same path.